components/color-palette/index.js

```diff
@@ -18,7 +18,7 @@ import { __, sprintf } from '@wordpress/i18n';
 import './style.scss';
 import Button from '../button';
 
-export default function ColorPalette( { colors, disableCustomColors = false, value, onChange } ) {
+export default function ColorPalette( { colors, disableCustomColors = false, value, onChange, disableAlpha = true } ) {
```

Comment:
A nitpick but it might be nice to sort these props alphabetically. The current ordering makes it harder to sort through arguments and makes this last one feel especially tacked-on.

Reply:
@tofumatt done!